<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mediaEditor</class>
 <widget class="QWidget" name="mediaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="photoLabel">
        <property name="text">
         <string>image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="photoButton">
        <property name="text">
         <string>Edit image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>autor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="yearLabel">
        <property name="text">
         <string>year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yearLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="durationLine"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="music">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>481</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="albumLabel">
             <property name="text">
              <string>album</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="albumLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lyricsLabel">
             <property name="text">
              <string>lyric</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="lyricsButton">
             <property name="text">
              <string>Edit lyrics</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="podcast">
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>481</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="episodeLabel">
             <property name="text">
              <string>episode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="episodeLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="seasonLabel">
             <property name="text">
              <string>season</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="seasonLine"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="audiobook">
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>481</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="readerLabel">
             <property name="text">
              <string>reader</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="readerLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="summaryLabel">
             <property name="text">
              <string>summary</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="summaryButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
